examples/cndpfwd/main.h

```diff
@@ -55,14 +55,15 @@ enum {
     FWD_ACL_STATS   = (1 << 4), /**< Enable printing ACL stats */
 };
 
-#define PKT_API_TAG    "pkt_api"         /**< Packet API json tag */
-#define NO_METRICS_TAG "no-metrics"      /**< json tag for no-metrics */
-#define NO_RESTAPI_TAG "no-restapi"      /**< json tag for no-restapi */
-#define ENABLE_CLI_TAG "cli"             /**< json tag to enable/disable CLI */
-#define MODE_TAG       "mode"            /**< json tag to set the mode flag */
-#define UDS_PATH_TAG   "uds_path"        /**< json tag for UDS to get xsk map fd */
-#define FIB_RULES_TAG  "l3fwd-fib-rules" /**< json tag to set up static FIB entries */
-#define HS_PATTERN_TAG "hs-patterns"     /**< json tag for Hyperscan patterns */
+#define PKT_API_TAG          "pkt_api"         /**< Packet API json tag */
+#define NO_METRICS_TAG       "no-metrics"      /**< json tag for no-metrics */
+#define NO_RESTAPI_TAG       "no-restapi"      /**< json tag for no-restapi */
+#define ENABLE_CLI_TAG       "cli"             /**< json tag to enable/disable CLI */
+#define MODE_TAG             "mode"            /**< json tag to set the mode flag */
+#define UDS_PATH_TAG         "uds_path"        /**< json tag for UDS to get xsk map fd */
+#define XSK_MAP_PIN_PATH_TAG "xsk_pin_path"    /**< json tag for pinned BPF map to get xsk map fd */
```

Comment:
In the other options we were using '-' not underscore. Can it be changed to xsk-pin-path?

Reply:
sure - I will update it